go-controller/pkg/config/config.go

```diff
@@ -470,6 +473,12 @@ var CommonFlags = []cli.Flag{
 		Usage:       "path of a file to direct log output to",
 		Destination: &cliConfig.Logging.File,
 	},
+	cli.Int64Flag{
+		Name:        "ovn-controller-resync-timer",
+		Usage:       "Periodic timer setting in seconds for ovn-controller to force recomputing flows (default 30)",
```

Comment:
@trozet how about "How often (in seconds) to force ovn-controller to resync flows (default 30)"